<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="flowTreeHorizontalLayout">
      <item>
       <widget class="QStackedWidget" name="stackedWidget"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1150</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="styleSheet">
    <string notr="true">QDockWidget {
    border: 4px solid red;    
}
</string>
   </property>
   <property name="windowTitle">
    <string>导航</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2">
     <item>
      <layout class="QVBoxLayout" name="navigationVerticalLayout"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="windowTitle">
    <string>设计模板</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="templateTreeHorizontalLayout"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_3">
   <property name="windowTitle">
    <string>属性编辑</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="propertyWidgetHorizontalLayout"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_4">
   <property name="windowTitle">
    <string>属性</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="navigationProptertyVerticalLayout"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNewTestProduct">
   <property name="text">
    <string>新建测试项</string>
   </property>
  </action>
  <action name="actionNewFlow">
   <property name="text">
    <string>新建流程</string>
   </property>
  </action>
  <action name="actionPase">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/icon/paste.png</normaloff>:/icon/paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>删除</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/icon/save.png</normaloff>:/icon/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/icon/copy.png</normaloff>:/icon/copy.png</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="actionNewProject">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/icon/new.png</normaloff>:/icon/new.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qrc.qrc"/>
 </resources>
 <connections/>
</ui>
